dom/coord.go

```diff
@@ -0,0 +1,6 @@
+package dom
+
+type Coord struct {
```

Comment:
maybe rename Coord ->  Location to align with function names and other parts of the code?